libcipher/transpos.py

```diff
@@ -0,0 +1,35 @@
+""" 
+
+Transposition cipher implementation.
+
+Provides functions to encrypt / decrypt a message by transpose ciphering it.
+
+Encryption needs a key(a number).
```

Comment:
See below for the docstring format and layout.

    """Single line description.
    
    Followed by a long description.
    """"